test/services/helpers/services/schoolGroups.js

```diff
@@ -14,7 +14,7 @@ const cleanup = async () => {
 	}
 	const ids = createdSchoolGroups;
 	createdSchoolGroups = [];
-	return schoolGroupModel.deleteMany({ _id: { $in: ids } });
+	return schoolGroupModel.deleteMany({ _id: { $in: ids } }).lean().exec();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5258109)